### DISCUSSION TITLE
How to set edge_index in torch_geometric.data.Data if all nodes are not connected

### DISCUSSION THREAD

**Discussion by thu1911** [pyg-team/pytorch_geometric#3337] [Q&A] [answered]
How to set _**edge_index**_ parameter in _**torch_geometric.data.Data**_ if all nodes are independent (not connected with any other nodes)?

**Comment by rusty1s** [comment, marked as answer]
You can create an empty Tensor of shape `[2, 0]`:
```python
edge_index = torch.tensor([[], []], dtype=torch.long)
```
